File: client/Client/Network/Messages/Editor/EnemySpawnPointRequestLockMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LoadRunnerClient.Network.Messages {
	class EnemySpawnPointRequestLockMessage {

		public const string TYPE = "EnemySpawnPointRequestLockMessage";

		private int _gridX, _gridY;

		public int gridX {
			get => _gridX;
			set => _gridX = value;
		}

		public int gridY {
			get => _gridY;
			set => _gridY = value;
		}

		public EnemySpawnPointRequestLockMessage(int gridX, int gridY) {
			this.gridX = gridX;
			this.gridY = gridY;		
		}

		public EnemySpawnPointRequestLockMessage() {
		}
	}
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/EnemySpawnPointRequestLockMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.dtos.EnemySpawnPointDTO;
import de.hsrm.mi.swtpro03.server.game.dtos.ItemDTO;

public class EnemySpawnPointRequestLockMessage {

	public static final String TYPE = "EnemySpawnPointRequestLockMessage";

	private int gridX, gridY;
	
	public EnemySpawnPointRequestLockMessage() {
	}

	public EnemySpawnPointRequestLockMessage(int gridX, int gridY) {	
		this.gridX = gridX;
		this.gridY = gridY;
	}

	public int getGridX() {
		return gridX;
	}

	public void setGridX(int gridX) {
		this.gridX = gridX;
	}

	public int getGridY() {
		return gridY;
	}

	public void setGridY(int gridY) {
		this.gridY = gridY;
	}

	
	
}
